tests/skydive_tests/README.md

```diff
@@ -79,3 +79,14 @@ To monitor additional pods (for example- Skydive pods that are deployed in diffe
 
 - Note 1: 
 To configure execution of tests, modify the file tests_conf.yaml and rerun. Full list of configration parameters can be found in  tests_conf_template.yaml
+
```

Comment:
Good ... so just the creation of those will cause in split to ingress and egress ... or there is configuration or something ... maybe you can create those directories, and put "template" example for "get_target_ip" file